miners/text/sample_config.txt

```diff
@@ -14,14 +14,14 @@ dendrite.max_worker_threads: 150
 dendrite.requires_grad: true
 dendrite.timeout: 5
 
-logging.debug: false
+logging.debug: true
```

Comment:
Should logging debug be true by default?

Reply:
logging should be turned off by default, but I left it on due the current bug where it doesn't display any of the subscription logs